executor/prepared.go

```diff
@@ -174,7 +212,7 @@ func (e *PrepareExec) Next(ctx context.Context, req *chunk.Chunk) error {
 		Params:        sorter.markers,
 		SchemaVersion: e.is.SchemaMetaVersion(),
 	}
-	prepared.UseCache = plannercore.PreparedPlanCacheEnabled() && plannercore.Cacheable(stmt)
+	prepared.UseCache = plannercore.PreparedPlanCacheEnabled() && plannercore.Cacheable(stmt) && planCacheHit
```

Comment:
Please raise warning when plan cache is disabled in configuration, while the hint indicates using plan cache. Or we'd better change the `USE_PLAN_CACHE(bool)` hint in parser to `DISABLE_PLAN_CACHE()`.

Reply:
OK，I will fix it.

Reply:
done